doc/integrator/multi_organization.rst.tmpl

```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
 - org2-registered
 - org2-intranet
 
+Internationalization
+--------------------
+
+We should have one localization file for each organization.
+
+The general workflow is:
+- The integrator will run `make update-client-po`.
+- This will run one `update-po` script for each organization with different environment variables.
+- We will get one po file set for each organization with the defined suffix.
+- The integrator needs to complete the po files with translations.
+- When we build the config Docker image it will convert all the po file in the corresponding JSON files
+  with the same suffix.
+- At runtime the `dynamic` view will return the appropriated URLs for the translations.
```

Comment:
At runtime, the `dynamic` view will return the appropriate URLs for the translations.